test/metabase/api/google_test.clj

```diff
@@ -0,0 +1,12 @@
+(ns metabase.api.google-test
+  (:require [clojure.test :refer :all]
+            [metabase.test :as mt]))
+
+(def ^:private test-client-id "test-client-id.apps.googleusercontent.com")
+
+(deftest google-settings-test
+  (testing "PUT /api/google/settings"
+    (testing "Valid Google Sign-In settings can be saved via an API call"
```

Comment:
These tests are currently just checking the status code, Shouldn't we test the setting value as well?